yaml-tests/src/test/resources/null-extraction-tests.yamsql

```diff
@@ -55,6 +55,55 @@ test_block:
 # subsequent runs of the same query will reuse the cached plan since the equality constraint
 # is satisfied when both arguments are `null`.
       - query: select * from B where !! !n null !! = 42 and !! !n null !! = 43
-      - explain: "COVERING(I1 <,> -> [B1: KEY[3], B2: KEY[1], B3: KEY[0]]) | FILTER promote(@c8 AS INT) EQUALS @c6 AND promote(@c12 AS INT) EQUALS @c6 | FETCH"
+      - explainContains: "COVERING(I1 <,> -> [B1: KEY[3], B2: KEY[1], B3: KEY[0]])"
```

Comment:
Is there a reason these are `explainContains` instead of `explain`? I've found `explainContains` can be harder to work with as it won't necessarily get fixed by the correction mode. Does this need to use `explainContains` because it's unstable which plan gets used or something due to the plan cache or something like that?

Reply:
I see. I figured that it might be something like that (that's what I meant by "it's unstable which plan gets used or something due to the plan cache"). It would probably good to have that written down in the code, as we've had other cases with similar bugs, and then been surprised later when we try to "fix" the test case and it is now broken